vispy/app/backends/_glfw.py

```diff
@@ -27,7 +27,7 @@
 try:
     import glfw
 except ImportError:
-    why_not = "Could not import glwf, you may need to `pip install glfw` first."
+    why_not = "Could not import glfw, you may need to `pip install glfw` first."
```

Comment:
:man_facepalming: It only took 8 years or so to find this typo. Nice catch and thank you.

Reply:
Haha, I forgot to install glfw and "glwf" popped up :p